init_test.go

```diff
@@ -20,21 +21,33 @@ var stack struct {
 	BuildImageID string
 	RunImageID   string
 }
+var localRegistryPort int
 
 func by(_ string, f func()) { f() }
 
+func getFreePort() (port int, err error) {
+	if l, err := net.Listen("tcp", ":0"); err == nil {
```

Comment:
I think it would be helpful to add a comment inside this method saying something like:

```
// Port `0` on Unix systems corresponds to a system-selected free port.
```